user_modules/shared_testcases/commonFunctions.lua

```diff
@@ -951,11 +961,190 @@ end
 --! @brief Check if DB contains column with data
 --! @param db_path path to DB
 --! @param sql_query contains select query with determine name of column. Don't use query with *
+--! example of sql_query: "SELECT preloaded_pt FROM module_config"
+--! "SELECT functional_group_id FROM app_group WHERE application_id='0000001'"
+--! "SELECT functional_group_id FROM app_group WHERE application_id='0000001'"
+--! "SELECT functional_group_id FROM app_group WHERE application_id=\\\"0000001\\\""
 --! @param exp_result contains data for comparing data from DB
 --! @return Returns false if expected data are not equal with DB data, otherwise returns true.
 function commonFunctions:is_db_contains(db_path, sql_query, exp_result)
   local column_db = commonFunctions:get_data_policy_sql(db_path, sql_query)
   return commonFunctions:is_table_equal(column_db, exp_result)
 end
 
+--------------------------------------------------------------------------------------------
+--20. Function checks PTU sequence need to call when PTU is triggered before call of function – it should just finalize PTU
+-- Checkes PTU HTTP flow.
+---------------------------------------------------------------------------------------------
+--! @brief Checks PTU HTTP flow sequence
+--! @param ptu_path path to PTU file
+--! @param ptu_name contains name of PTU file
+function commonFunctions:check_ptu_sequence_partly(self, ptu_path, ptu_name)
+  check_file_existing(ptu_path)
+  local CorIdSystemRequest = self.mobileSession:SendRPC("SystemRequest",
+    {
+      requestType = "HTTP",
+      fileName = ptu_name,
+    },ptu_path)
+  EXPECT_HMICALL("BasicCommunication.SystemRequest"):Times(0)
```

Comment:
@LuxoftAKutsan I see reason because within PROPRIETARY flow SDL sends SystemRequest to HMI
for HTTP flow SDL doesn't send SystemRequest I checked this here